python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_restraint.py

```diff
@@ -755,6 +777,11 @@ def getCorrection(self, method="analytical"):
             correction can introduce errors when the restraints are weak,
             restrained angles are close to 0 or pi radians, or the restrained
             distance is close to 0.
+        flavour : str
+            When analytical correction is used, one could either use
+            Boresch's derivation or Schrödinger's derivation. Both of
```

Comment:
Maybe explicitly give the allowed values? E.g. `schrodinger` instead of `Schrödinger`